travis_configure.sh

```diff
@@ -117,8 +117,6 @@ run_tests()
 {
     if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
```

Comment:
what about "Don't follow `the` log `output` from libdrizze-redux container" as a commit title (if it isn't too long)

Also there is a typo in `and` in the commit message. s/and/an